quiz solution/quiz-2.ipynb

```diff
@@ -0,0 +1,177 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "95810a2b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "6438a3a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f=open('contacts.csv','w',newline='')"
```

Comment:
write in a other way to open file ?

Reply:
with open('contacts.csv','w',newline=' ') as f: